dapr/serializers/json.py

```diff
@@ -28,6 +27,9 @@ def serialize(
             custom_hook: Optional[Callable[[object], bytes]] = None) -> bytes:
 
         dict_obj = obj
+
+        # importing this from top scope creates a circular import
+        from dapr.actor.runtime.config import ActorRuntimeConfig
```

Comment:
Wait. Why we did not have this before?

Reply:
We did, I just moved it out of the module scope.

Reply:
You mean we had a circular import before?